### DISCUSSION TITLE
Help for compiling using native image

### DISCUSSION THREAD

**Discussion by VegetableDog118** [oracle/graal#5489] [Q&A]
I have already set the classpath and java path but when I try to use the native image to compile the helloworld class I got the following error:

<img width="1392" alt="compile_issue" src="https://user-images.githubusercontent.com/65802837/203644011-e3f9373a-3d7b-41c3-9dc7-ab905219a31b.png">

Any help would be appreciated.

**Comment by olpaw** [comment]
@VegetableDog118 which version of GraalVM are you using? Can you reproduce this issue with the latest version of GraalVM
https://www.graalvm.org/downloads/
